src/main/java/com/pubnub/api/endpoints/objects_api/utils/IncludeParamValue.java

```diff
@@ -0,0 +1,43 @@
+package com.pubnub.api.endpoints.objects_api.utils;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.pubnub.api.endpoints.objects_api.utils.Include.INCLUDE_PARAM_NAME;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class IncludeParamValue implements ParameterEnricher {
```

Comment:
IMHO if you check my other comment for CompositeParameterEnricher this class is not needed at all. There's everything already in place

Reply:
ok